docs/_posts/2016-01-11-installation-guide.md

```diff
@@ -374,15 +374,16 @@ above and have WAR files for both MISO (`ROOT.war`) and Run Scanner(`runscanner-
 To install or upgrade, perform the following steps:
 
 1. Backup your existing database.
-1. Stop Tomcat.
-1. Remove `$CATALINA_HOME/webapps/ROOT`.
+1. Stop MISO's Tomcat.
+1. `rm -rf $CATALINA_HOME/webapps/ROOT*`.
```

Comment:
I'd prefer clarifying that the entire directory and the .war should both be removed, rather than providing a `rm -rf` command to copy/paste without thought